README.md

```diff
@@ -1,4 +1,45 @@
 # pyroptosis_signature_data_analysis
 Analysis Repository for the Interstellar project. For data processing for this project please see [pyroptosis_signature](https://github.com/WayScience/pyroptosis_signature)
 
+### Workflow:
+
+Cells were incubated for 1h in the inhibitors, and then incubated for 6h in each inducer.
+For more information on plate layout and experimental design please see [pyroptosis_signature](https://github.com/WayScience/pyroptosis_signature).
+
+
+# Treatments of Interest
+
+| Treatments        | Number of Doses   | Function                      |
+| :----------       | :---------------: | :---------                    |
+| Media             | 1                 | Control                       |
+| DMSO              | 1                 | Vehicle Control               |
+| Thapsigargin      | 2                 | ER Stress Induced Apoptosis   |
+| H202              | 2                 | Cell Death                    |
+| Flagellin         | 3                 | Induce Pyroptosis             |
+| LPS               | 5                 | Induce Pyroptosis             |
+| LPS + Nigericin   | 2 * 3 = (6)       | Induce Pyroptosis             |
+| Disulfiram        | 3                 | Pyroptosis Inhibitor          |
+
+
+# Other Treatments and combinations
+
+| Treatments        | Number of Doses   | Inhibitors | Number of Doses | Function                      |
```

Comment:
Now this took me a bit to understand.

For example, above, when you mention the `treatments of interest`, you state the doses of LPS+Nigercin are 6 in total, but then you now have 2 different LPS+Nigercin doses below this.

I am assuming this just means that out of the 6 total combination of doses, there is only one of those combinations with this inhibitor.

Does it make sense to mention what combination of doses this is specifically? Is it a high+high, low+high, etc.?

I can see this being confusing to someone who doesn't know this dataset as well.

Another example I could see is that above you have three doses total for Flagellin, but here you only have **two** doses used in combination with an inhibitor. Where did the other dose go?